packages/runtime-core/src/apiOptions.ts

```diff
@@ -249,6 +283,9 @@ export function applyOptions(
     for (const key in computedOptions) {
       const opt = (computedOptions as ComputedOptions)[key]
 
+      checkDuplicateProperties &&
```

Comment:
should still use `__DEV__ && checkDuplicateProperties!(...)` to drop the line completely in production.